api/api.go

```diff
@@ -376,7 +384,61 @@ func (m *ApiService) handleValidatorOffchainStateByIndex(w http.ResponseWriter,
 	})
 }
 
+func (m *ApiService) handleValidatorRelayers(w http.ResponseWriter, req *http.Request) {
+	vars := mux.Vars(req)
+	valPubKey := vars["valpubkey"]
+	if !IsValidPubkey(valPubKey) {
+		m.respondError(w, http.StatusInternalServerError, fmt.Sprintf("invalid validator pubkey format"))
+		return
+	}
+	var relays = []string{
```

Comment:
Can we move all this to `config.go`? And then use the `network` field to choose the relay set?

You would need to make a minor modification in `ApiService` struct, including the `Config` field. So that you can `m.Config.Network` or something like that.

And very important, we should differenciate between goerli and mainnet relayers. I have the feeling that here they are mixed (eg relay.ultrasound.money is for mainnet right?)

In other words, we need to know if the validator is registered in a goerli relayer OR mainnet relayer, depending on the network we are running. If mainnet, we only chain mainet relayer, if goerli we only check goerli relayers. We should not mix

Reply:
relays changed to config.go. 2 sets, one for mainnet and one for goerli. 

ApiService now has a new field "Network", initialized at "NewApiService" method. The endpoint handler of /registeredrelays/{pubkey} now checks the Network field to choose between the two relay sets.

Related to this, I did a minor extra change at config.go, setting "mainnet" as a default value when -network flag is not provided by user.